style.css

```diff
@@ -0,0 +1,9 @@
+form {
+	padding-top: 100px;
+}
+
+#contact-list {
+	list-style-type: none;
+    margin-left: -3.75rem;
```

Comment:
Instead of adding margin-left, you can do
```html
<div class="medium-6 medium-offset-1">
```
instead of 
```html
<div class="medium-4 medium-offset-4">
```
Around line 24